src/main/java/com/example/SOPSbackend/service/DoctorService.java

```diff
@@ -33,7 +48,52 @@ public void addVaccinationSlot(DoctorEntity doctor, Instant date) {
             throw new InternalValidationException(Map.of("date", "Invalid date value"));
 
         VaccinationSlotEntity newSlot = new VaccinationSlotEntity(doctor, transformedDate);
-        vaccinationSlotRepository.save(newSlot);
+        //if(vaccinationSlotRepository.findResultsByDate(transformedDate).stream().findAny().orElse(null) != null)
```

Comment:
To w końcu jak?

Reply:
To tak
Chciałem, żeby sprawdzało czy mam taką datę w bazie, ale to wywalało testy z jakiegoś powodu, a nie miałem czasu żeby już szukać